spec/meilisearch/index/settings_spec.rb

```diff
@@ -680,4 +682,51 @@ def update_synonyms(index, synonyms)
 
     client.wait_for_task(task['taskUid'])
   end
+
+  context 'On pagination sub-routes' do
+    let(:index) { client.index(uid) }
+    let(:pagination) { { maxTotalHits: 3141 } }
+
+    before { client.create_index!(uid) }
+
+    it 'gets default values of pagination' do
+      settings = index.pagination.transform_keys(&:to_sym)
+      expect(settings).to be_a(Hash)
+      expect(settings).to eq(default_pagination)
+    end
```

Comment:
I think you don't need to test if it is a Hash, because your `transform_keys` will break before the test :)

```suggestion
    it 'gets default values of pagination' do
      settings = index.pagination.transform_keys(&:to_sym)

      expect(settings).to eq(default_pagination)
    end
```